webapp/src/modules/mortgage/utils.js

```diff
@@ -21,11 +30,39 @@ export function getLoanMetadata() {
   return `#mortgage #required-cosigner:${mortgageManagerContract.address}`
 }
 
-export function getActiveMortgagesByBorrower(mortgages, borrower) {
-  return mortgages.filter(
-    mortgage =>
-      mortgage &&
-      mortgage.borrower === borrower &&
-      mortgage.status !== MORTGAGE_STATUS.cancelled
+export function isMortgageActive(mortgage, parcel) {
+  return (
+    mortgage &&
+    parcel &&
+    ((isMortgagePending(mortgage) &&
+      isOpen(parcel.publication, PUBLICATION_STATUS.open)) ||
+      hasStatus(mortgage, [MORTGAGE_STATUS.ongoing, MORTGAGE_STATUS.paid]))
+  )
+}
+
+/**
+ * Returns mortgages active -> pending status require an open publication
+ * @param {array} - mortgages
+ * @param  {array} - parcels
+ * @returns {array} - mortgages
+ */
+export function getActiveMortgages(mortgages, parcels) {
+  return mortgages
```

Comment:
if mortgages could be undefined, maybe instead of adding a `?:` we ca use:

`export function getActiveMortgages(mortgages = [], parcels = {}) {`

or 

export function getActiveMortgages(mortgages, parcels) {
  mortgages = mortgages || []